pgsql/pointcloud.sql.in

```diff
@@ -186,6 +186,10 @@ CREATE OR REPLACE FUNCTION pcpoint_in(cstring)
 	RETURNS pcpoint AS 'MODULE_PATHNAME', 'pcpoint_in'
 	LANGUAGE 'c' IMMUTABLE STRICT;
 
+CREATE OR REPLACE FUNCTION pcpoint_in(bytea)
```

Comment:
I wouldn't name this function `pcpoint_in`. `pcpoint_in` and `pcpoint_out` are the `pcpoint` type's input and output functions, respectively. So I'd rather not add a `pcpoint_in` function that doesn't participate to the type definition.

So I suggest that we name this function `PC_PointFromWKB`, which is consistent with PostGIS' `ST_GeomFromWKB` function.